src/mlpack/core/data/split_data.hpp

```diff
@@ -52,7 +54,8 @@ void Split(const arma::Mat<T>& input,
            arma::Mat<T>& testData,
            arma::Row<U>& trainLabel,
            arma::Row<U>& testLabel,
-           const double testRatio)
+           const double testRatio,
+           bool shuffleData = true)
```

Comment:
```suggestion
           const bool shuffleData = true)
```
(Just for consistency with the other parameters. :))

There are a couple places where this comment applies, but I've only highlighted the first.

Reply:
Done